pkg/apis/core/validation/validation.go

```diff
@@ -2816,6 +2816,45 @@ func validatePodResourceClaimSource(claimSource core.ClaimSource, fldPath *field
 	return allErrs
 }
 
+func validateLivenessProbe(probe *core.Probe, fldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if probe == nil {
+		return allErrs
+	}
+	allErrs = append(allErrs, validateProbe(probe, fldPath)...)
+	if probe.SuccessThreshold != 1 {
```

Comment:
I don't remember liveness probe has any limitations on success threshold. Only startup probes has this strange limitation

Reply:
taking it back. Strange